[2024-08-16T22:05:11.850+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-16T22:05:11.860+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: predict_values_TS_5.get_datasets_from_google scheduled__2024-08-04T00:00:00+00:00 [queued]>
[2024-08-16T22:05:11.865+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: predict_values_TS_5.get_datasets_from_google scheduled__2024-08-04T00:00:00+00:00 [queued]>
[2024-08-16T22:05:11.865+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-16T22:05:11.875+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_datasets_from_google> on 2024-08-04 00:00:00+00:00
[2024-08-16T22:05:11.880+0000] {warnings.py:110} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=2365) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-16T22:05:11.881+0000] {standard_task_runner.py:63} INFO - Started process 2369 to run task
[2024-08-16T22:05:11.881+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'predict_values_TS_5', 'get_datasets_from_google', 'scheduled__2024-08-04T00:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmp4rguuurs']
[2024-08-16T22:05:11.882+0000] {standard_task_runner.py:91} INFO - Job 12: Subtask get_datasets_from_google
[2024-08-16T22:05:11.913+0000] {task_command.py:426} INFO - Running <TaskInstance: predict_values_TS_5.get_datasets_from_google scheduled__2024-08-04T00:00:00+00:00 [running]> on host 08898c2f83c5
[2024-08-16T22:05:11.967+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Ruslan Solarev' AIRFLOW_CTX_DAG_ID='predict_values_TS_5' AIRFLOW_CTX_TASK_ID='get_datasets_from_google' AIRFLOW_CTX_EXECUTION_DATE='2024-08-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-04T00:00:00+00:00'
[2024-08-16T22:05:11.968+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-16T22:05:27.669+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-16T22:05:27.670+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-16T22:05:27.676+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=predict_values_TS_5, task_id=get_datasets_from_google, run_id=scheduled__2024-08-04T00:00:00+00:00, execution_date=20240804T000000, start_date=20240816T220511, end_date=20240816T220527
[2024-08-16T22:05:27.704+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-16T22:05:27.713+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-16T22:05:27.716+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
